components/browser/awesomebar/src/main/java/mozilla/components/browser/awesomebar/layout/DefaultSuggestionViewHolder.kt

```diff
@@ -79,6 +84,9 @@ internal sealed class DefaultSuggestionViewHolder {
                 editView.visibility = View.GONE
             } else {
                 editView.visibility = View.VISIBLE
+
+                if (usingBottomBrowserToolbar) editView.rotation = EDIT_ARROW_ROTATION
```

Comment:
nit: braces around the if-statement even though it's a one liner since this is an assignment to a different attribute.